### DISCUSSION TITLE
Персональная логика НПС

### DISCUSSION THREAD

**Discussion by IamUnloaded** [OGSR/OGSR-Engine#429] [Q&A]
В оригинале, персональная логика при спавне НПС, перезапускается с начала. Здесь это фиксили каким-то способом?

**Comment by xrSimpAI** [comment]
Логика нпс полностью управляется скриптами. К движку этот вопрос отношения не имеет. И как это правится я не помню.

**Comment by IamUnloaded** [comment]
В этом я пока углубленно не разбирался, но все-же готов утверждать, что после спавна движок(и никто иной) определяет куда передать управление НПСом. Или я не прав?

**Comment by xrSimpAI** [comment]
Ты просто не понимаешь как это работает. Изучи скрипты xr_motivator, xr_logic для начала.
